controllers/validate.go

```diff
@@ -0,0 +1,83 @@
+// Package controllers is the layer for imput date in the application.
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// DockerRequestPayloadNew is sent by docker hub whenever a new push happen to a
+// repository.
+type DockerRequestPayloadNew struct {
+	CallbackURL string `json:"callback_url"`
```

Comment:
Please add your changes to the original `DockerRequestPayload` struct instead (in docker.go).

Reply:
okay